com.unity.netcode.gameobjects/Runtime/Messaging/Messages/DestroyObjectMessage.cs

```diff
@@ -89,20 +81,27 @@ public bool Deserialize(FastBufferReader reader, ref NetworkContext context, int
             ByteUnpacker.ReadValueBitPacked(reader, out NetworkObjectId);
             if (networkManager.DistributedAuthorityMode)
             {
-                reader.ReadByteSafe(out m_DestroyFlags);
+                reader.ReadByteSafe(out byte bitset);
```

Comment:
I think it's best if we read the flags before the if check? It seems to be the same logic regardless